wordpress-mobile/WordPress-Android
# Support dark mode in Zendesk chat

Fixes #21485 

Previously Zendesk chat would show with light mode colors even in dark mode. This PR addresses this.

To test:

* Enable dark mode either in the app or the device
* Me > Help & support > Contact Support
* Verify the chat screen uses dark colors
* Disable dark mode and verify the chat screen uses light colors

**Note:** Dark mode support is only available in Android 33 (Tiramisu) and above.

![dark](https://github.com/user-attachments/assets/07024f23-a3cf-4fcf-a13d-23b279e548a2)
